aspnetcore/migration/proper-to-2x/index.md

```diff
@@ -10,10 +10,10 @@ ms.topic: article
 ms.assetid: 3155cc9e-d0c9-424b-886c-35c0ec6f9f4e
 ms.technology: aspnet
 ms.prod: asp.net-core
-uid: migration/mvc2
+uid: migration/proper-to-2x/index
```

Comment:
In the future, don't change UID.  The convention is the UID is generated from the path to create a unique UID. That's just a convention.